File: CS/Solution5/Solution5.Module.Win/MyBarActionItemsFactoryProvider.cs
using DevExpress.ExpressApp.Win.Templates;
using DevExpress.XtraBars;

namespace Solution5.Module.Win
{
    public class MyBarActionItemsFactoryProvider : IBarActionItemsFactoryProvider
    {
        public BarActionItemsFactory CreateBarActionItemsFactory(BarManager barManager)
        {
            return new MyBarActionItemsFactory(barManager);
        }
    }
}

File: VB/Solution5/Solution5.Module.Win/MyBarActionItemsFactoryProvider.vb
Imports Microsoft.VisualBasic
Imports System
Imports DevExpress.ExpressApp.Win.Templates
Imports DevExpress.XtraBars

Namespace Solution5.Module.Win
	Public Class MyBarActionItemsFactoryProvider
		Implements IBarActionItemsFactoryProvider
        Public Function CreateBarActionItemsFactory(ByVal barManager As BarManager) As BarActionItemsFactory Implements IBarActionItemsFactoryProvider.CreateBarActionItemsFactory
            Return New MyBarActionItemsFactory(barManager)
        End Function
	End Class
End Namespace
